CPUMeter.c

```diff
@@ -187,6 +187,15 @@ static void AllCPUsMeter_getRange(const Meter* this, int* start, int* count) {
    }
 }
 
+static void AllCPUsMeter_updateValues(Meter* this) {
+   CPUMeterData* data = this->meterData;
+   Meter** meters = data->meters;
+   int start, count;
+   AllCPUsMeter_getRange(this, &start, &count);
+   for (int i = 0; i < count; i++)
```

Comment:
Should this loop start from 'start' and not zero?

Reply:
`meters` holds single core meters in the range [0..count-1] with the ids [start..start+count-1]